### DISCUSSION TITLE
20241204_101203_vlan12_firewall.pcap filename in kibana

### DISCUSSION THREAD

**Discussion by Anelka3523** [Security-Onion-Solutions/securityonion#13993] [Unsupported Versions] [answered]
I have files named 20241204_101203_vlan12_firewall.pcap that I need to process with the so-import-pcap script. i want to search these file names by “filename” pcap file name when querying kibana. how can i add pcap file names to *.log files with zeek output? is there a setting in the zeek program? or do i need to make a setting in the elasticsearch pipeline?

**Comment by InfosecGoon** [comment, marked as answer]
The filename is not preserved by the import script -- the traffic is replayed as if it were being observed by the monitoring interface.
